packages/theme-a11y/theme-a11y.js

```diff
@@ -199,3 +199,127 @@ export function removeTrapFocus() {
+ * @param {object} messages - Custom messages to overwrite with keys: newWindow, external, newWindowExternal
+ * @param {string} messages.newWindow - When the link opens in a new window (e.g. target="_blank")
+ * @param {string} messages.external - When the link is external (e.g. www.cnn.com)
+ * @param {string} messages.newWindowExternal - When the link is external and opens in a new window (<a href="http://www.shopify.com" target="_blank">Shopify</a>)
+ * @param {object} targetLinks - Specific elements to be targeted
+ * @param {object} prefix - Prefix to namespace "id" of the messages
+ */
+export function accessibleLinks(messages, targetLinks, prefix) {
+  var accessibleLinksHandlers = {};
+  var messagesSource = messages || {};
+  var body = document.querySelector('body');
+  var prefixName = prefix || 'a11y';
+  var idSelectors = {
+    newWindow: prefixName + '-new-window-message',
+    external: prefixName + '-external-message',
+    newWindowExternal: prefixName + '-new-window-external-message'
+  };
+
+  if (targetLinks === undefined || targetLinks === null) {
+    targetLinks = document.querySelectorAll('a[href]:not([aria-describedby])');
+  }
+
+  if (targetLinks !== '' && typeof targetLinks !== 'object') {
+    throw new TypeError(targetLinks + ' is not an Element object.');
+  }
+
+  accessibleLinksHandlers.generateHTML = function(customMessages) {
+    if (typeof customMessages !== 'object') {
+      customMessages = {};
+    }
+
+    var localMessages = Object.assign(
+      {
+        newWindow: 'Opens in a new window.',
+        external: 'Opens external website.',
+        newWindowExternal: 'Opens external website in a new window.'
+      },
+      customMessages
+    );
+
+    var container = document.createElement('ul');
+    var htmlMessages = '';
+
+    for (var message in localMessages) {
+      htmlMessages +=
+        '<li id=' +
+        idSelectors[message] +
+        '>' +
+        localMessages[message] +
+        '</li>';
+    }
+
+    container.setAttribute('hidden', true);
+    container.innerHTML = htmlMessages;
+
+    body.appendChild(container);
+  };
+
+  accessibleLinksHandlers.externalSite = function(link) {
+    return link.hostname !== window.location.hostname;
+  };
+
+  targetLinks.forEach(function(link) {
+    var target = link.getAttribute('target');
+    var rel = link.getAttribute('rel');
+    var isExternal = accessibleLinksHandlers.externalSite(link);
+    var isTargetBlank = target === '_blank';
+
+    if (isExternal) {
+      link.setAttribute('aria-describedby', idSelectors.external);
+    }
+
+    if (isTargetBlank) {
+      if (rel === null || rel.indexOf('noopener') === -1) {
+        link.setAttribute('rel', 'noopener');
+      }
+
+      link.setAttribute('aria-describedby', idSelectors.newWindow);
+    }
+
+    if (isExternal && isTargetBlank) {
+      link.setAttribute('aria-describedby', idSelectors.newWindowExternal);
+    }
```

Comment:
Delete above conditions and add them as else ifs below:

```suggestion
    } else if (isExternal) {
      link.setAttribute('aria-describedby', idSelectors.external);
    } else if (isTargetBlank) {
      if (rel === null || rel.indexOf('noopener') === -1) {
        link.setAttribute('rel', 'noopener');
      }
       link.setAttribute('aria-describedby', idSelectors.newWindow);
    }
```